test/test_query2.cpp

```diff
@@ -6160,4 +6164,26 @@ TEST(Query_FullTextMulti)
     CHECK(table->get_search_index(col)->is_empty());
 }
 
+TEST(Query_FullTextPrefix)
```

Comment:
could you add a few tests that check a prefix search as well as a normal word and an exclude? Does an exclude override a prefix match?

Reply:
I added a test. Was that what you meant? Do you have further suggestions?